frontend/src/components/common/Resource/index.test.ts

```diff
@@ -39,7 +39,7 @@ function getFilesToVerify() {
   const filesToVerify: string[] = [];
   fs.readdirSync(__dirname).forEach(file => {
     const fileNoSuffix = file.replace(/\.[^/.]+$/, '');
-    if (!avoidCheck.find(suffix => fileNoSuffix.endsWith(suffix))) {
+    if (fileNoSuffix && !avoidCheck.find(suffix => fileNoSuffix.endsWith(suffix))) {
```

Comment:
this change seems to be unrelated to the websockets?

Reply:
Will add it in a separate commit